.circleci/config.yml

```diff
@@ -58,22 +58,51 @@ commands:
             ./deps/readies/bin/getpy3
             ./deps/readies/bin/getpy2
 
+  save-tests-logs:
+    steps:
+      - run:
+          name: Cleanup test log dir
+          command: |
+            if [[ -d tests/flow/logs ]]; then
+              cd tests/flow/logs
+              rm -f *.aof *.rdb
+            fi
+            if [[ -d tests/tck/logs ]]; then
+              cd tests/tck/logs
+              rm -f *.aof *.rdb
+            fi
+          when: always
+      - store_artifacts:
+          path: tests/flow/logs
+      - store_artifacts:
+          path: tests/tck/logs
+
   persist-artifacts:
     steps:
+      - early-return-for-forked-pull-requests
+      - run:
+          name: List artifacts
+          command: |
+            cd bin/artifacts
+            du -ah --apparent-size *
       - persist_to_workspace:
-          root: bin/artifacts
+          root: bin/
           paths:
-            - 'snapshots/*.zip'
-            - 'snapshots/*.tgz'
-            - '*.zip'
-            - '*.tgz'
+            - artifacts/snapshots/*.zip
+            - artifacts/snapshots/*.tgz
+            - artifacts/snapshots/*.tar
+            - artifacts/*.zip
+            - artifacts/*.tgz
+            - artifacts/*.tar
 
   setup-prerequisits:
     steps:
       - setup-automation
       - run:
           name: Install prerequisites
-          command: ./sbin/system-setup.py
+          command: |
+            ./sbin/system-setup.py
+            python3 -m pip list
```

Comment:
Why do we list installed python packages ?

Reply:
Just for information/disgnostics, not essential, can be removed.